vue-custom-bootstrap/src/components/FormSection.vue

```diff
@@ -1,12 +1,14 @@
 
 <template>
+  <div :id="divId" class="container-fluid">
```

Comment:
I think you can get rid of `:id="divId"` here. By default it should apply the id to the outer div anyways.

Reply:
divId is the value that links a table of contents item to its corresponding FormSection, so it can't be removed as it is